docs/concepts/tasks.md

```diff
@@ -53,6 +55,29 @@ Tasks are uniquely identified by a task key, which is a hash composed of the tas
 
     To be clear, there's nothing stopping you from putting all of your code in a single task &mdash; Prefect will happily run it! However, if any line of code fails, the entire task will fail and must be retried from the beginning. This can be avoided by splitting the code into multiple dependent tasks.
 
+**Calling a task from another task**
```

Comment:
I like the idea of showing how to call a task's function, but the wording of calling a task from another task could be a little misleading since in the end you only run `my_second_task` which calls `my_first_task`'s function but it doesn't trigger its task run -- this could get confusing in the UI 
![image](https://user-images.githubusercontent.com/86264395/185796124-e08a342b-fb3f-42c9-9432-fcf2d90baafc.png)

I think we could keep it but phrase it as "Calling a task's function from another task" and add a note sth like:

Note that this way you are only calling the task's function without actually generating a task run. It has a negative effect on your observability - Prefect won't track task execution in your Prefect backend if you call it this way.